package.json

```diff
@@ -58,6 +58,7 @@
   },
   "devDependencies": {
     "@react-native-community/datetimepicker": "^6.1.2",
+    "@shopify/flash-list": "^1.4.1",
```

Comment:
This is another native dependency. If we don't care about sticky headers, the props are almost identical with the built-in React Native list.

So, we could either add this to the readme as yet another native dependency for people to add, or we could simply switch to the built-in list.